go-controller/pkg/ovn/egressip_local_test.go

```diff
@@ -106,12 +106,12 @@ var _ = Describe("Local gateway mode EgressIP Operations with", func() {
 				fakeOvn.fakeExec.AddFakeCmd(
 					&ovntest.ExpectedCmd{
 						Cmd:    fmt.Sprintf("ovn-nbctl --timeout=15 --data=bare --format=table --no-heading --columns=options find logical_router name=GR_%s options:lb_force_snat_ip!=-", node2.name),
-						Output: fmt.Sprintf("lb_force_snat_ip=%s", nodeGatewayRouterIP),
+						Output: fmt.Sprintf("lb_force_snat_ip=%s", nodeLogicalRouterIPv6),
```

Comment:
unrelated fix? I guess you were just using the wrong IP before and this just makes the test more correct but isn't actually fixing a bug?

Reply:
You are right. I've split this out into its own commit. I'm keeping the commit in this PR however as the test I added, checking the handler deletion, benefits from it.